### PR TITLE
Next

### PR DESCRIPTION
**Pull request by agbogdan**
payload proxy support was added and finally tested on the hummercloud job

### CODE REVIEW

**Comment by PalNilsson** [comment]
Closing this for now since it is blocking the pilot release (will reopen later today)

**Comment by PalNilsson** [comment]
Opening again..

**Comment by PalNilsson** [comment]
Thanks. Please change proxyContents to proxy_contents to be PEP compliant.

File: pilot/user/atlas/container.py [hunk 1]
```diff
@@ -224,6 +253,14 @@ def alrb_wrapper(cmd, workdir, job=None):
         # do not include the X509_USER_PROXY in the command the container will execute
         x509 = os.environ.get('X509_USER_PROXY')
```
Comment (PalNilsson) [inline pilot/user/atlas/container.py]: Please change this to os.environ.get('X509_USER_PROXY', '')

File: pilot/user/atlas/container.py [hunk 2]
```diff
@@ -20,10 +20,39 @@
...
+        if res is None:
+            logger.error("Unable to get proxy with role '%s' from panda server" % voms_role)
+            return False
+
+        if res['StatusCode'] != 0:
+            logger.error("When get proxy with role '%s' panda server returned: %s" % (voms_role, res['errorDialog']))
+            return False
+
+        proxy_contents = res['userProxy']
+        _outfile = open(proxy_outfile_name, 'w')
```
Comment (PalNilsson) [inline pilot/user/atlas/container.py]: Can you please use the write_file() function from pilot.util.filehandling here? (it's already imported)

Comment (agbogdan) [reply pilot/user/atlas/container.py]: Ok, done.

**Comment by PalNilsson** [comment]
Closing for now since it is blocking a pilot release

**Pull request closed without merging**